kitsune/questions/cron.py

```diff
@@ -72,7 +72,8 @@ def auto_archive_old_questions():
 
         cursor = connection.cursor()
         cursor.execute(sql)
-        transaction.commit_unless_managed()
+        if not transaction.get_connection().in_atomic_block:
```

Comment:
I think what you want is to surround the cursor code with `with transaction.atomic()` rather than this, since this doesn't actually start a transaction.